.github/workflows/current.yml

```diff
@@ -0,0 +1,53 @@
+name: Test on current Node.js
+
+on:
+  schedule:
+    - cron: '00 00 * * Mon'
```

Comment:
This is (intended to be) run at 00:00 UTC on Mondays; that's 4 am Pacific / 7 am Eastern. Happy to choose another time if that doesn't work for folks!

Reply:
Although it turns out using weekday names is a Vixie-ism that isn't part of the POSIX standard that GitHub uses; fixed to be `1` instead of `Mon`.